FMS/diag_table.json

```diff
@@ -9,13 +9,12 @@
       "minLength": 1
     },
     "base_date": {
-      "type": "array",
-      "minItems": 6,
-      "maxItems": 6,
-      "items": {
-        "type": "integer",
-        "minimum": 0
-      }
+      "type": "string",
+      "minLength": 1,
```

Comment:
Just a minor nitpick. `"minlength": 1` could possibly be confusing to someone reading the schema as documentation, since `$baseDate` is 9 characters and the shortest value allowed by the pattern is 11 characters.

Reply:
ah yes that is a good point. I removed `"minlength": 1`.